e2e/atlas/ldap_test.go

```diff
@@ -167,16 +154,117 @@ func TestLDAP(t *testing.T) {
 	})
 
 	t.Run("Delete", func(t *testing.T) {
+		testLDAPDelete(t, cliPath, g.projectID)
+	})
+}
+
+func TestLDAPWithStdin(t *testing.T) {
+	g := newAtlasE2ETestGenerator(t)
+	g.generateProjectAndCluster("ldap")
+
+	cliPath, err := e2e.Bin()
+	require.NoError(t, err)
+
+	var requestID string
+
+	t.Run("Verify", func(t *testing.T) {
 		cmd := exec.Command(cliPath,
 			atlasEntity,
 			securityEntity,
 			ldapEntity,
-			"delete",
+			"verify",
+			"--hostname",
+			ldapHostname,
+			"--port",
+			ldapPort,
+			"--bindUsername",
+			"cn=admin,dc=example,dc=org",
 			"--projectId", g.projectID,
-			"--force")
-		cmd.Env = os.Environ()
-		resp, err := cmd.CombinedOutput()
-		require.NoError(t, err, string(resp))
-		assert.Contains(t, string(resp), "LDAP configuration userToDNMapping deleted")
+			"-o",
+			"json")
+
+		passwordStdin := bytes.NewBuffer([]byte(ldapBindPassword))
+		cmd.Stdin = passwordStdin
+
+		requestID = testLDAPVerifyCmd(t, cmd)
 	})
+
+	require.NotEmpty(t, requestID)
+
+	t.Run("Save", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			atlasEntity,
+			securityEntity,
+			ldapEntity,
+			"save",
+			"--hostname",
+			ldapHostname,
+			"--port",
+			ldapPort,
+			"--bindUsername",
+			"cn=admin,dc=example,dc=org",
+			"--mappingMatch",
+			"(.+)@ENGINEERING.EXAMPLE.COM",
+			"--mappingSubstitution",
+			"cn={0},ou=engineering,dc=example,dc=com",
+			"--projectId", g.projectID,
+			"-o",
+			"json",
+		)
+
+		passwordStdin := bytes.NewBuffer([]byte(ldapBindPassword))
+		cmd.Stdin = passwordStdin
+
+		testLDAPSaveCmd(t, cmd)
+	})
+
+	t.Run("Delete", func(t *testing.T) {
+		testLDAPDelete(t, cliPath, g.projectID)
+	})
+}
+
+func testLDAPDelete(t *testing.T, cliPath string, projectID string) {
```

Comment:
```suggestion
func testLDAPDelete(t *testing.T, cliPath, projectID string) {
```

Reply:
thanks @andreaangiolillo . Addressed in latest.